tests/rest_api/test_get_list.py

```diff
@@ -0,0 +1,22 @@
+import requests
+import pytest
+
+BASE_URL = "https://drb-api-qa.nypl.org"
+
+def test_get_collection():
+
+    url = BASE_URL + "/collection/list?sort=title"
+    response = requests.get(url)
+
+    assert response.status_code is not None
+    assert response.status_code == 200, (
+        f"API call failed.\n"
+        f"Expected status code: {200}\n"
+        f"Actual status code: {response.status_code}\n"
+        f"URL: {url}\n"
+        f"Response text: {response.text[:100]}..."
+    )
+
+    response_json = response.json()
+    
+    assert response_json is not None
```

Comment:
nit: new line at eof

Reply:
do you always want to keep an empty line at the end? or no lines

Reply:
always a new line that is empty at the end of the file!